core/templates/dev/head/collection_player/collection_player.html

```diff
@@ -0,0 +1,89 @@
+{% extends "base.html" %}
+
+{% block subtitle %}
+  {{ collection_title }}
+{% endblock subtitle %}
+
+{% block header_js %}
+  {{ super() }}
+  {% if collection_id %}
+    <script type="text/javascript">
+      GLOBALS.collectionId = JSON.parse('{{collection_id|js_string}}');
+      GLOBALS.isLoggedIn = Boolean(JSON.parse('{{is_logged_in|js_string}}'));
+    </script>
+
+    <style>
+      html, body {
+        background-color: #eee;
+      }
+    </style>
+  {% endif %}
+
+  {{dependencies_html}}
+{% endblock header_js %}
+
+{% block navbar_breadcrumb %}
+  <ul class="nav navbar-nav oppia-navbar-breadcrumb">
+    <li>
+      <span class="oppia-navbar-breadcrumb-separator"></span>
+      <span class="protractor-test-collection-header" itemprop="description">{{collection_title}}</span>
```

Comment:
Yeah, I was wondering that, especially because we have things like 
Oppia > About 
(which is the about page, not the About exploration)
and
Oppia > Profile > AmitD

Maybe it should be:
Oppia > Explore > Welcome to Oppia
And the gallery would be:
Oppia > Explore